BUFRCREX_CodeFlag_en_03.csv

```diff
@@ -9,7 +9,9 @@ FXY,ElementName_en,CodeFigure,EntryName_en,EntryName_sub1_en,EntryName_sub2_en,N
 003001,Surface station type,7,Land station (local network),,,,Operational
 003001,Surface station type,8,Land vehicle,,,,Operational
 003001,Surface station type,9,Autonomous marine vehicle,,,,Operational
-003001,Surface station type,10-30,Reserved for future use,,,,Operational
+003001,Surface station type,10,Reserved for future use,,,,Operational
```

Comment:
10, Reserved

Reply:
@DavidBerryNOC  -- Is 10 supposed to be reserved?

Reply:
I'm not aware of what is has been reserved for if it is. It might be that we had reserved it for the ocean gliders (https://github.com/wmo-im/BUFR4/issues/16) but if we did we have then failed to include it in the glider sequence.

Reply:
ok.... so now we have 10 reserved and 12-30 reserved. 11 is the new code figure for 'tag attached to marine animal"

Reply:
I've gone back through emails and can't find anywhere where we have asked for 10 to be reserved for gliders (I would have thought that gliders would come under autonomous marine vehicles thinking on it).

I've also gone through the old non-operational versions to try and find if/where 10 was reserved but haven't been able to find anywhere.

I'm at a loss as to where the "10, Reserved for future use" came from and why I went straight to 11. If possible, it would make sense to use 10 rather than 11. I don't know is this is possible.